tests/fixed/trailing_comma_on_array.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types=1);
+
+$array =  [
```

Comment:
Now that I noticed, we don't have a rule to force only 1 space after the assignment operator 🤔

Reply:
ha! good point, I didn't notice that I've put 2 of them here

Reply:
I didn't notice that as well :scream: